BUILD

```diff
@@ -1301,3 +1301,14 @@ exports_files([
     "conformance/text_format_failure_list_java.txt",
     "conformance/text_format_failure_list_java_lite.txt",
 ])
+
+filegroup(
+    name = "bzl_srcs",
+    srcs = glob(["**/*.bzl"]),
+    visibility = ["//visibility:public"],
+)
+
+exports_files(
```

Comment:
This would allow anyone to depend on them in `load` statements.

Reply:
No, [exports_files](https://docs.bazel.build/versions/main/be/functions.html#exports_files) allows files in a package to be treated as targets. Nothing prevents loading a public symbol from any `.bzl` file.

Reply:
Ah right, this is a Bazel specific thing I always forget.

Reply:
https://docs.bazel.build/versions/main/visibility.html#visibility-of-bzl-files